proto/proto-backplane-grpc/src/main/proto/deephaven/proto/table.proto

```diff
@@ -276,9 +276,11 @@ service TableService {
 
 message TableReference {
   oneof ref {
-    // an exported ticket
+    // A ticket. In the case of a single-shot operation, this will often be an export id ticket.
```

Comment:
I'd go further, "A ticket to resolve to get the table. To handle races correctly, we encourage this to be an export ticket, but technically any ticket type will suffice." or something like that?

Reply:
I like it.